### DISCUSSION TITLE
lower case for Table names and Column names when working with Oracle Database

### DISCUSSION THREAD

**Discussion by mschrading** [doctrine/doctrine2#8694] [General]
In my case I have had a given Oracle Database which all there names Tables columns etc. are named in lower case. So when I had a mapping:import dotrine let write all names quoted which doen't work.

My question is does doctrine handle oracle Databases which lower case names in general?

Thanks for help.

Michael

**Comment by beberlei** [comment]
@mschrading Yes you can set them quoted using ticks, ``` @Column(name="`lower`") ``` - this is automatically transmitted into the right syntax for Oracle.
